pkg/cluster/config/v1alpha2/doc.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha2 implements the v1alpha2 apiVersion of the `kind` Config
+// that introduces multi node support
+//
+// +k8s:deepcopy-gen=package
+// +k8s:conversion-gen=sigs.k8s.io/kind/pkg/cluster/config
+// +k8s:defaulter-gen=TypeMeta
+package v1alpha2
```

Comment:
maybe it's time to start adding docs with Node config examples.

Reply:
yes, let's document this once it's ready for use, and until then keep it to the config example file, for those that want to do some digging.